storage/chunks_queue.go

```diff
@@ -0,0 +1,13 @@
+package storage
+
+import (
+	"github.com/onflow/flow-go/model/chunks"
+)
+
+type ChunksQueue interface {
+	StoreChunkLocator(locator *chunks.Locator) (bool, error)
```

Comment:
I would suggest we define a cetralized chunkId similar to locator and reuse it everywhere, but def not in the scope of this PR